docs/en/setup/README.md

```diff
@@ -0,0 +1,19 @@
+# Setup
+First and most important thing is, SkyWalking Satellite startup behaviours are driven by configs/satellite_config.yaml. Understood the setting file will help you to read this document.
+
+## Startup script
+The startup script is /bin/skywalking-satellite-{version}-{plateform}-amd64. 
+1. Start SkyWalking Satellite.
+```shell script
+./bin/skywalking-satellite-{version}-{plateform}-amd64 start --config=./configs/satellite_config.yaml
```

Comment:
This is correct and not correct at the same time. This actually is a dev-command for the source code local compiling startup.
But usually, like you see in SkyWalking's main repo release, you should have a DIST tar, including binaries and config file in the same folder. 
@EvanLjp Do you plan to use another PR to do this?

Reply:
please ignore it, would add the start script in another PR for release.